rebuild/src/public_key.cc

```diff
@@ -259,7 +259,7 @@ PublicKeyArg::PublicKeyArg(Napi::Env env)
       _public_key{nullptr},
       _bytes{_env} {};
 
-PublicKeyArg::PublicKeyArg(Napi::Env env, Napi::Value raw_arg)
+PublicKeyArg::PublicKeyArg(Napi::Env env, const Napi::Value &raw_arg)
```

Comment:
There are a few places that `Napi::Value raw_arg` need to be updated to `const Napi::Value &raw_arg`.  There will need to be a final pass on the rebuild branch once the rest of the pieces get merged to touch up